source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -49,9 +50,28 @@ class Application(Document):
 class BlockQuote(Ia2Web):
 	role = controlTypes.ROLE_BLOCKQUOTE
 
-class Editor(Ia2Web):
+class Editor(Ia2Web, DocumentWithTableNavigation):
 	TextInfo = MozillaCompoundTextInfo
 
+	def _getTableCellAt(self,tableID,startPos,destRow,destCol):
+		obj=startPos.NVDAObjectAtStart
+		while not obj.table and obj!=self:
+			obj=obj.parent
+		if not obj.table:
+			raise LookupError
+		table = obj.table
+		try:
+			try:
+				cell = table.IAccessibleTable2Object.cellAt(destRow - 1, destCol - 1).QueryInterface(IAccessibleHandler.IAccessible2)
+			except AttributeError:
+				cell = table.IAccessibleTableObject.accessibleAt(destRow - 1, destCol - 1).QueryInterface(IAccessibleHandler.IAccessible2)
+			cell = IAccessible(IAccessibleObject=cell, IAccessibleChildID=0)
+			if cell.IA2Attributes.get('hidden'):
+				raise LookupError("Found hidden cell") 

```

Comment:
How should this be handled? Is this left over debugging? If its not debugging, perhaps a comment to say what this might mean for those catching the exception.